scripts/uberenv/packages/serac/package.py

```diff
@@ -89,8 +86,12 @@ class Serac(CMakePackage, CudaPackage):
     variant("devtools",  default=False,
             description="Build development tools (such as Sphinx, AStyle, etc...)")
 
+    variant('caliper', default=False, 
+            description='Build with hooks for Caliper performance analysis')
     variant('glvis', default=False,
             description='Build the glvis visualization executable')
+    variant('netcdf', default=False,
```

Comment:
How long does this take to build? I'm thinking this would be nice to have on by default.

Reply:
on toss3... clang took 14min, gcc took 16min.  There are a whole set of libraries that crept in.  I can look to see if i can utilize some system libraries to keep them down.  I'll look into that after I have the blueos build working again.

Reply:
What broke on blueos?

Reply:
* mvapich compiler wrapper couldnt compile anything
* the python path change caused the devtools variant to go crazy because i was comparing the root path of python and doxygen.

Reply:
If it takes that long I think we should leave it as optional.

Reply:
That was total time.  I havent timed netcdf by itself.

Reply:
Ahhh OK. Then I think it's a good idea to have it required.